platforms/mixin-common/README.md

```diff
@@ -1,4 +1,4 @@
 # mixin-common
 
-This project contains mixins shared between Forge, Fabric and Quilt, as
+This project contains mixins shared between Forge, Fabric and formerly Quilt, as
```

Comment:
is "and formerly quilt" needed? (if it is, also use an oxford comma, so: "Forge, Fabric, and formerly Quilt")

Reply:
yes because I intend to keep the quilt module in tree for now